samples/nrf9160/modem_shell/Kconfig

```
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic Modem Shell config"
+
+config MOSH_IPERF3
+	bool  "Enable iperf3 support"
+	default y
+	select NRF_IPERF3
+
+config MOSH_SOCK
+	bool  "Enable socket tool support"
+	default y
+
+config MOSH_PING
+	bool  "Enable ICMP ping support"
```

```suggestion
	bool  "ICMP ping support"
```

also missing help text